linter/transform_import_tools.py

```diff
@@ -0,0 +1,15 @@
+
+import astroid
+
+def register(_):
+    pass
+
+def transform_tools(module):
+    """ Transform import module """
+    if 'cross_building' in module.locals:
+        del module.locals['cross_building']
```

Comment:
The problem with this change is that it raises a `no-member` error, that's all. I cannot (I don't know how to) add a message to suggest an alternative import for that `cross_building` function.

Reply:
Can't you do here a `self.add_message()` instead of doing a `del`? I am totally unexperienced with this, so I am surely missing something

Reply:
There is no `self` here, this is not a `BaseChecker`. Here I'm transforming the imported module, I don't even know if the `cross_building` is being used at all, I'm just _removing_ it from the module. Then the linter raises if it is used somewhere, because the member doesn't exist.

Reply:
Why? What do you expect? 

The alternative would be to visit the AST and post the error from the node where `tools.cross_building` is being used. Easier to add a message, and harder to code. I can investigate that approach.